components/PostVisitActionInput.tsx

```diff
@@ -1,4 +1,7 @@
-import { Checkboxes as LBHCheckboxes } from "lbh-frontend-react/components";
+import {
+  Checkboxes as LBHCheckboxes,
+  List,
+} from "lbh-frontend-react/components";
 import { Textarea } from "lbh-frontend-react/components/Textarea";
```

Comment:
Next time you come across this sort of thing, combine them into a single import statement.